src/components/SecurityPage.jsx

```diff
@@ -5,19 +5,41 @@ import { translate } from 'cozy-ui/transpiled/react/I18n'
 import { withClient } from 'cozy-client'
 import flowRight from 'lodash/flowRight'
 import { Link } from 'react-router-dom'
+import Wrapper from 'components/Wrapper'
+import NarrowContent from 'cozy-ui/transpiled/react/NarrowContent'
+import strongPasswordIcon from 'assets/strong-password.svg'
+import Stack from 'cozy-ui/transpiled/react/Stack'
+import { MainTitle, Text, Bold } from 'cozy-ui/transpiled/react/Text'
+import Card from 'cozy-ui/transpiled/react/Card'
+import { UnorderedList, ListItem } from 'cozy-ui/transpiled/react/UnorderedList'
 
 const DumbLinkToSettings = withClient(props => {
   // eslint-disable-next-line no-unused-vars
   const { client, t, f, ...rest } = props
   const cozyURL = new URL(client.getStackClient().uri)
   const { cozySubdomainType } = client.getInstanceOptions()
   const settingsAppSlug = 'settings'
-  const settingsAppHref = generateWebLink({
+  const rawSettingsAppHref = generateWebLink({
     cozyUrl: cozyURL.origin,
     slug: settingsAppSlug,
     subDomainType: cozySubdomainType
   })
 
+  const passwordsUrl = generateWebLink({
+    cozyUrl: cozyURL.origin,
+    slug: 'passwords',
+    subDomainType: cozySubdomainType
+  })
+
+  const successUrl = passwordsUrl + 'installation'
+  const cancelUrl = passwordsUrl + 'security/hint'
```

Comment:
Nit: a bit brittle to build URLs like this, could you use the URL constructor here ?

Reply:
See https://github.com/cozy/cozy-passwords/pull/1/commits/71406fce3d564a5206870975e9c83dec4ceb16f1